### ISSUE TITLE
Fix new behavior of pa11y-ci 3.1.0

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#2386]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Since pa11y-ci has been bumped from 3.0.2 to 3.1.0, the accessibility job fails (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/runs/6969221673/job/18964696038?pr=2385).

This is probably due to the peer dependency that makes also a bump from pa11y `~6.1.0` to `^6.2.3` (to analyze).

It can be reproduced locally too by running:
```
$ npm ci
$ npm run dist
$ npm run release-sri
$ npm run docs-build
$ npm run docs-accessibility

Errors in http://localhost:9001/docs/5.3/forms/checks-radios/:

 • ARIA hidden element must not contain focusable elements
   (https://dequeuniversity.com/rules/axe/4.2/aria-hidden-focus?application=axeAPI)

   (html > body > div:nth-child(6) > main > div:nth-child(3) >
   div:nth-child(65) > div:nth-child(1) > form > fieldset)

   <fieldset class="star-rating" disabled="" aria-hidden="true"> <legend
   class="visually-hi...</fieldset>

✘ 131/132 URLs passed
ERROR: "docs-pa11y" exited with 2.
Error: Process completed with exit code 1.
```

The aim of this issue is to fix either the use case or to tell pa11y not to scan the use cases (by modifying `.pa11y-ci.json`) which corresponds to https://boosted.netlify.app/docs/5.3/forms/checks-radios/#disabled-2.

Note: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/888db1ea0c45e6c9bf07490ff57017c2692ba16a hid temporarily this use case to continue working on the dark mode: `fieldset.star-rating[disabled]`.

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Boosted are you using?

main (v5.3.3)
